tests/specs/functional/helpers-spec.k

```diff
@@ -0,0 +1,41 @@
+requires "abi.md"
+
+module VERIFICATION
+    imports EVM-ABI
+
+    syntax Step ::= Bool | Bytes
+ // ------------------------------
+
+    syntax KItem ::= runLemma ( Step ) [symbol(runLemma)]
+                   | doneLemma( Step )
+ // --------------------------------------
+    rule <k> runLemma( T ) => doneLemma( T ) ... </k>
+
+endmodule
+
+module HELPERS-SPEC
+    imports VERIFICATION
+
+
+ // EVM Type helper
+ // ---------------
+
+    // checking the function evaluation that went wrong in LLVM (issue booster-558)
+    claim [isStaticType-fix-2371]:
+//      <k> runLemma ( #isStaticType( #bytes(b"\229\230/\221") ) ) => doneLemma ( false ) ... </k>
```

Comment:
```suggestion
      <k> runLemma ( #isStaticType( #bytes(b"\229\230/\221") ) ) => doneLemma ( true ) ... </k>
```

Any harm in adding this test too?

Reply:
Well, the result should still be `false` since it is `#bytes` (no fixed length). The syntax `\229` is wrong, I would have to use `\xe6\xe5\xdd` but I can add that test. Or delete the comment.

Reply:
Added the test - pushing as soon as the `--use-booster` run of the integration tests is finished.